front/app/components/IdeaCard/Footer/ReadMoreButton.tsx

```diff
@@ -6,10 +6,11 @@ import messages from '../messages';
 import { useIntl } from 'utils/cl-intl';
 
 interface Props {
-  slug: string;
+  slug?: string;
+  onClick?: () => void;
```

Comment:
Here also, I would not add an additional `onClick` handler but manage the scroll to top behaviour in the component.